src/main/java/org/broadinstitute/gatk/nativebindings/pairhmm/PairHMMNativeArguments.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.gatk.nativebindings.pairhmm;
+
+/**
+ * Struct used to pass arguments to the PairHMM computation.
+ */
+public class PairHMMNativeArguments {
```

Comment:
Should we keep this as a "struct" type class with public mutable fields, or make the fields final and set them via a constructor? Thoughts?